test/gradient/ast_specifier_test.exs

```diff
@@ -1294,78 +1294,276 @@ defmodule Gradient.AstSpecifierTest do
             ]} = recv
   end
 
+  test "typespec when" do
+    {tokens, ast} = load("/Elixir.TypespecWhen.beam", "/typespec_when.ex")
```

Comment:
A leading slash suggests that we use an absolute path, but we clearly do not, as this file belongs to the project and is therefore stored relative to the project's directory. Could we make `load/2` take that into account? This might be done outside this PR, as it's not strictly relevant to this its functionality.

Reply:
Ok, I'll change it